File: GlassDoor/ViewModels/SeedAngular.cs
using DAL.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace GlassDoor.ViewModels
{
    public class SeedAngular
    {
        public ICollection<JobType> jobTypes  { get; set; }
        public ICollection<JobCategory> jobCategories { get; set; }
        public ICollection<Country> countries { get; set; }
        public ICollection<City> cities { get; set; }
        public ICollection<CareerLevel> careerLevels { get; set; }
        public ICollection<Currency> Currencies { get; set; }
        public ICollection<SalaryRate> salaryRates { get; set; }
        public ICollection<Skill> Skills { get; set; }
    




    }
}

File: GlassDoor/ClientApp/src/app/modules/company/models/seed-angular.ts
import { CareerLevels } from "./career-levels";
import { Cities } from "./cities";
import { Countries } from "./countries";
import { Currencies } from "./currencies";
import { educationLevels } from "./education-levels";
import { JobCategories } from "./job-categories";
import { JobTypes } from "./job-types";
import { SalaryRates } from "./salary-rates";
import { Skills } from "./skills";

export class SeedAngular {
    constructor(public cities:Cities[],public careerLevels:CareerLevels[], public countries:Countries[],public currencies:Currencies[],public jobCategories:JobCategories[],public jobTypes:JobTypes[],public salaryRates:SalaryRates[],public skills:Skills[]
    ,public educationLevels:educationLevels[]){
    }
}
